lib/mapping/CMapEditManager.cpp

```diff
@@ -331,21 +343,31 @@ void CComposedOperation::execute()
 
 void CComposedOperation::undo()
 {
-	for(auto & operation : operations)
+	//reverse order
+	for(auto & operation = operations.rbegin(); operation != operations.rend(); operation++)
```

Comment:
There is compilation error pointing to this line (see actions). Maybe removing & will help

Reply:
done